### DISCUSSION TITLE
How can I capture a comment?

### DISCUSSION THREAD

**Discussion by couriourc** [ast-grep/ast-grep#472] [Q&A] [answered]
```javascript
/**
* It's a comment
*/
```
`It's a comment`,which is I want to get.

**Comment by HerringtonDarkholme** [comment, marked as answer]
You can first use `kind: comment` and then use regular expression.
`ast-grep` has a `--json` flag.

```
      --json
          Output matches in structured JSON text useful for tools like jq. Conflicts with interactive
```

https://ast-grep.github.io/playground.html#eyJtb2RlIjoiQ29uZmlnIiwibGFuZyI6ImphdmFzY3JpcHQiLCJxdWVyeSI6ImNvbnNvbGUubG9nKCRNQVRDSCkiLCJyZXdyaXRlIjoibG9nZ2VyLmxvZygkTUFUQ0gpIiwiY29uZmlnIjoicnVsZTpcbiAga2luZDogY29tbWVudFxuICBwYXR0ZXJuOiAkQSIsInNvdXJjZSI6Ii8qKlxuKiBJdCdzIGEgY29tbWVudFxuKi8ifQ==
